src/CodeGeneration/ApiGenerator/Configuration/CodeConfiguration.cs

```diff
@@ -41,7 +42,15 @@ public static class CodeConfiguration
 			// these APIs are new and need to be mapped
 			"ml.set_upgrade_mode.json",
 			"ml.find_file_structure.json",
-			"monitoring.bulk.json"
+			"monitoring.bulk.json",
+			"snapshot.cleanup_repository.json",
+			"ml.estimate_memory_usage.json",
+			"indices.clone.json",
+
+			"slm.delete_lifecycle.json",
+			"slm.execute_lifecycle.json",
+			"slm.get_lifecycle.json",
+			"slm.put_lifecycle.json",
```

Comment:
These are marked as stable, so should be implemented in the 7.4 release (OK to be ignored for now)

Reply:
yes, they're stable but don't want to conflate generating implementations for them in this PR 🙂